lynkctx.py

```diff
@@ -298,7 +298,7 @@ def download(self):
         variables = {
             "envId": self.env_id,
             "sbomId": self.ver_id,
-            "includeVulns": False
```

Comment:
@viveksahu26 this needs to a param. We should not always download vulns

Reply:
Yeah, added !!